pkg/controller/flow/flow.go

```diff
@@ -43,6 +44,7 @@ import (
 	// TODO: Get rid of these, but needed as other controllers use them.
 	servingclientset "github.com/knative/serving/pkg/client/clientset/versioned"
 	servinginformers "github.com/knative/serving/pkg/client/informers/externalversions"
+	servingconfigmaps "github.com/knative/serving/pkg/configmap"
```

Comment:
I'm totally fine for using this from the serving stack for now, but we should pull this into a common place so we don't have to take a dependency between repos. Perhaps add a TODO here and file an issue in Serving to move this out to some common repo.

Reply:
It is now in `knative/pkg`